gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/EncassLinker.java

```diff
@@ -27,6 +42,14 @@ public void link(Encass encass, Bundle bundle, Bundle targetBundle) {
         if (policy == null) {
             throw new LinkerException("Could not find policy for Encapsulated Assertion: " + encass.getName() + ". Policy ID: " + encass.getPolicyId());
         }
+
+        String portalTemplate = "false";
+        try {
```

Comment:
Thank you for making this more cleaner with the help of a simplifier.
Move this try-catch to the simplifier.

Reply:
Done